### PR TITLE
updating 1.0.0 to 2.0.0

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/2424

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ Review the known issues for Red Hat Advanced Cluster Management for Kubernetes.
 [#certificate-manager-must-not-exist-during-an-installation]
 === Certificate manager must not exist during an installation
 
-// 1.0.0:678
+// 2.0.0:678
```
Comment (swopebe) [inline release_notes/known_issues.adoc]: the one leave as-is for known issue strategy

File: troubleshoot_acm/master.html [hunk 2]
```diff
@@ -530,7 +530,7 @@ <h4 id="resolving-the-problem-multiclusterhub-operator-start">Resolving the prob
 <ul>
 <li>
 <p>An incorrect snapshot format was entered.
-The correct format is <code>1.0.0-SNAPSHOT-2020-04-01-02-43-31</code>.
+The correct format is <code>2.0.0-SNAPSHOT-2020-04-01-02-43-31</code>.
```
Comment (swopebe) [inline troubleshoot_acm/master.html:533]: I am not sure what this file is used for and we need to make sure that is the actual name of the image.

**Review by swopebe — COMMENTED** [review]
looks good, thank you.

Couple small comments.

**Review by swopebe — APPROVED** [review]

File: troubleshoot_acm/master.html (hunk 2 above)
Comment (dockerymick) [reply troubleshoot_acm/master.html:533]: I will make a note to ensure that we update image names: https://github.com/open-cluster-management/backlog/issues/2424#issuecomment-662640824

**Pull request merged**